web/server/codechecker_server/api/report_server.py

```diff
@@ -503,7 +504,7 @@ def unzip(b64zip, output_dir):
     """
     This function unzips the base64 encoded zip file. This zip is extracted
     to a temporary directory and the ZIP is then deleted. The function returns
-    the name of the extracted directory.
+    the size of the extracted zip file.
```

Comment:
The function returns the size of the zip file not the size of the extracted data rigth?